doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,30 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html) iceoryx tries to keep things simple and offers only basic Quality of Service (QoS) settings. The QoS settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings, which are also available in DDS, are:
```

Comment:
They are not directly available in DDS I would say, it is not immediately obvious how the concepts/options map to each other. I would leave DDS out here.

In DDS we have a `history depth` which roughly corresponds  to our `queueCapacity` on subscriber side. However, in DDS this is also available on the writer, which roughly corresponds to our `historyCapacity`. It also depends on the setting Durability setting, late joining readers in DDS only get values with `Non-Volatile` durability, e.g. `TRansient Local`.

So it is not straightforward to map the concepts, and for this reason I would leave it out or phrase it differently and mention that similar settings exist for DDS (durability and history depth QoS).

Reply:
Ok, I reformatted the list to a table.